### ISSUE TITLE
What changes would be necessary in a production environment

### ISSUE THREAD

**Issue by urumat** [chatwoot/rasa-agent-bot-demo#13]
Hello, I'm planning to run this code in a production environment. Some of the changes I made were

- I change the conversation status to 'Open' when the user wants to talk to a human
- I check the status of the conversation and only forward to RASA when the status is 'Pending'.

Are there any other changes that would be good to have in a production environment?
Is there any performance upgrade I should consider making?

Thanks in advance,
Matias.

**Comment by sujanpgowda** [comment]
Rasa needs to be made a first class integration like dialogflow, out of the box, for it to be used in production, like dialogflow already is.

**Comment by sojan-official** [comment]
@matiasperes It should be okay to to be run as it is.

@sujanpgowda it's on our roadmap 
https://github.com/chatwoot/chatwoot/issues/3597

**Comment by sujanpgowda** [comment]
@sojan-official Yes, we are aware of that. But we had to switch to csml studio for the rasa integration.
